api/can_api_definition.py

```diff
@@ -75,6 +76,14 @@ class CovidPatientsMethod(enum.Enum):
     ESTIMATED = "estimated"
 
 
+class TestPositivityRatioMethod(GetByValueMixin, enum.Enum):
+    """Method used to determine test positivity ratio."""
+
+    HHSTesting = "HHSTesting"
+    VALORUM = "Valorum"
```

Comment:
Valorum feels pretty unhelpful for anyone who doesn't know the internals of our API data pipeline. I guess there's not really much of a better option right now